### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: src/NodaTime/Calendars/HebrewScripturalCalculator.cs [hunk 1]
```diff
@@ -171,17 +171,11 @@ internal static int GetDaysFromStartOfYearToStartOfMonth(int year, int month)
...
-            4 => 29,
-            6 => 29,
-            10 => 29,
-            13 => 29,
+            var m when m == 2 || m == 4 || m == 6 || m == 10 || m == 13 => 29,
             8 => IsHeshvanLong(year) ? 30 : 29,
             9 => IsKislevShort(year) ? 29 : 30,
             12 => IsLeapYear(year) ? 30 : 29,
-            _ => 30 // 1, 3, 5, 7, 11, 13
+            _ => 30 // 1, 3, 5, 7, 11
```
Comment (Youssef1313) [inline src/NodaTime/Calendars/HebrewScripturalCalculator.cs]: This was possibly a comment typo ?
the `13` case were already handled directly.

Comment (jskeet) [reply src/NodaTime/Calendars/HebrewScripturalCalculator.cs]: Yes, this was a typo. I'm fine for there to be a typo fix.

File: src/NodaTime/Calendars/HebrewScripturalCalculator.cs [hunk 2]
```diff
@@ -171,17 +171,11 @@ internal static int GetDaysFromStartOfYearToStartOfMonth(int year, int month)
...
         internal static int DaysInMonth(int year, int month) => month switch
         {
-            // FIXME: How do we express multiple cases in a switch expression?
-            // We want: (2, 4, 6, 10, 13) => 29,
-            2 => 29,
-            4 => 29,
-            6 => 29,
-            10 => 29,
-            13 => 29,
+            var m when m == 2 || m == 4 || m == 6 || m == 10 || m == 13 => 29,
```
Comment (jskeet) [inline src/NodaTime/Calendars/HebrewScripturalCalculator.cs]: I'd really rather not do this. I think it's less clear, and may well be less efficient too.

Comment (Youssef1313) [reply src/NodaTime/Calendars/HebrewScripturalCalculator.cs]: Well. I'll revert it.
A cleaner way will be added to C# 9.0.

**Comment by jskeet** [comment]
Please rebase this into a single change, with an appropriate description ("Remove typo from comment" or similar.)

I'm very keen on having a commit history which is clear and easy to read.

**Comment by Youssef1313** [comment]
> 
> 
> Please rebase this into a single change, with an appropriate description ("Remove typo from comment" or similar.)
> 
> I'm very keen on having a commit history which is clear and easy to read.

Ok, I'll clone it locally first because GitHub doesn't allow to do so.
But I'm just wondering why not use the "squash" option when merging. It produces a single commit.

**Comment by jskeet** [comment]
Yes, I *could* squash instead - but I think it's generally better for the person contributing the change to make it as clean as possible, minimizing the work for the maintainer to do. There's work that only the maintainer can do - and useful input that only the maintainer can provide, such as reasons for doing things a particular way. The more effort a contributor makes in terms of presenting their contribution cleanly, the more time the maintainer has for that.

**Pull request merged**